lib/charliecloud.py

```diff
@@ -599,10 +608,21 @@ def exit(code):
 
 def init(cli):
    # logging
-   global log_festoon, log_fp, trace_fatal, verbose
+   global log_festoon, log_fp, log_quiet, trace_fatal, verbose
+   log_quiet = cli.quiet
    assert (0 <= cli.verbose <= 3)
    verbose = cli.verbose
    trace_fatal = (cli.debug or bool(os.environ.get("CH_IMAGE_DEBUG", False)))
+   if (log_quiet > 0):
+      fail_ct = 0
+      if (trace_fatal):
+         ERROR("“quiet” incompatible with “debug”")
```

Comment:
Might want to mention the environment variable here too since folks might not make the translation or remember they set it.